### PR TITLE
Homework added

### PR DESCRIPTION
**Pull request by SilentKingdom13**
Homework completed. Still getting to grips with Github though, so apologies if I've missed any steps with it.

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,36 @@ class Program
...
+            // Their BMI
+            float heightConvert1 = height1 / 100;
+            float height_s1 = heightConvert1 * heightConvert1;
+            float sum1 = weight1 / height_s1;
+            Console.WriteLine(name1 + " has a BMI of " + sum1 + ".");
+
+
+            // Second person = variable2
+            string name2 = "Constance Derpleton";
+            int age2 = 24;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'age2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834&bid=16054140)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,36 @@ class Program
...
+
+            // Their BMI
+            float heightConvert1 = height1 / 100;
+            float height_s1 = heightConvert1 * heightConvert1;
+            float sum1 = weight1 / height_s1;
+            Console.WriteLine(name1 + " has a BMI of " + sum1 + ".");
+
+
+            // Second person = variable2
+            string name2 = "Constance Derpleton";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'name2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,36 @@ class Program
...
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Homework Lesson 1 & 2 - SilentKingdom
+            // BMI Calculator for 2 people
+
+            // First person = variable1
+            string name1 = "Tom Jefferson";
+            int age1 = 19;
+            float weight1 = 50f;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weight1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,36 @@ class Program
...
+            float height1 = 156.5f;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'height1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Homework Lesson 1 & 2 - SilentKingdom
+            // BMI Calculator for 2 people
+
+            // First person = variable1
+            string name1 = "Tom Jefferson";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'name1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

File: Src/BootCamp.Chapter/Program.cs [hunk 6]
```diff
@@ -6,7 +6,36 @@ class Program
...
+            float height_s1 = heightConvert1 * heightConvert1;
+            float sum1 = weight1 / height_s1;
+            Console.WriteLine(name1 + " has a BMI of " + sum1 + ".");
+
+
+            // Second person = variable2
+            string name2 = "Constance Derpleton";
+            int age2 = 24;
+            float weight2 = 37f;
+            float height2 = 128.4f;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'height2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

File: Src/BootCamp.Chapter/Program.cs [hunk 7]
```diff
@@ -6,7 +6,36 @@ class Program
...
+            float heightConvert1 = height1 / 100;
+            float height_s1 = heightConvert1 * heightConvert1;
+            float sum1 = weight1 / height_s1;
+            Console.WriteLine(name1 + " has a BMI of " + sum1 + ".");
+
+
+            // Second person = variable2
+            string name2 = "Constance Derpleton";
+            int age2 = 24;
+            float weight2 = 37f;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weight2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

File: Src/BootCamp.Chapter/Program.cs [hunk 8]
```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Homework Lesson 1 & 2 - SilentKingdom
+            // BMI Calculator for 2 people
+
+            // First person = variable1
+            string name1 = "Tom Jefferson";
+            int age1 = 19;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'age1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834&bid=16054140)

File: Src/BootCamp.Chapter/Program.cs [hunk 9]
```diff
@@ -6,7 +6,42 @@ class Program
...
+
+            // Second person = variable2
+            const string name2 = "Tom Jefferson";
+            const int age2 = 19;
+            const float weight2 = 50f;
+            const float height2 = 156.5f;
+            Console.WriteLine(name2 + " is " + age2 + ", and weighs " + weight2 + "kg at a height of " + height2 + "cm.\n");
+
+            // Their BMI
+            float heightConvert2 = height2 / 100;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightConvert2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)

**Comment by SilentKingdom13** [comment]
I didn't understand the need for 2 people if it required user input (besides updating variables after a copy and paste below), hence my original code simply printing 2 examples without user input, then calculating BMI. I opted for a mix of input and the print example for my own reference really.

**Comment by ellmaster** [comment]
@Almantask 
Clarification needed.

File: Src/BootCamp.Chapter/Program.cs [hunk 10]
```diff
@@ -6,7 +6,42 @@ class Program
...
+            Console.Write("How much do you weigh (kg)? ");
+            float weight1 = float.Parse(Console.ReadLine());
+            Console.Write("How tall are you? (cm) ");
+            float height1 = float.Parse(Console.ReadLine());
+            // Testing string + variable layouts
+            Console.WriteLine("\n{0} is {1}, and weighs {2}kg at a height of {3}cm.\n", name1, age1, weight1, height1);
+
+            // Their BMI
+            float heightConvert1 = height1 / 100;
+            float height_s1 = heightConvert1 * heightConvert1;
```
Comment (ignasposka) [inline Src/BootCamp.Chapter/Program.cs]: It's not super important, but you might want to take a look at variable naming conventions in C# https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md

File: Src/BootCamp.Chapter/Program.cs [hunk 11]
```diff
@@ -6,7 +6,42 @@ class Program
...
+            float sum1 = weight1 / height_s1;
+            Console.WriteLine(name1 + " has a BMI of " + sum1 + ".");
+
+            // Comparison to Tom Jefferson
+            Console.WriteLine("\nWe'll compare " + name1 + "'s BMI with that of a previous subject:\n");
+
+            // Second person = variable2
+            const string name2 = "Tom Jefferson";
```
Comment (ignasposka) [inline Src/BootCamp.Chapter/Program.cs]: Read second person information from the console as you did for the first one.

File: Src/BootCamp.Chapter/Program.cs (hunk 10 above)
Comment (SilentKingdom13) [reply Src/BootCamp.Chapter/Program.cs]: Thanks for that, I'll try to stick with naming conventions in the future.

**Pull request closed without merging**